R/drive_mkdir.R

```diff
@@ -54,9 +55,18 @@ drive_mkdir <- function(path = NULL, name = NULL, verbose = TRUE) {
 
   path <- path %||% root_folder()
   path <- as_dribble(path)
-  confirm_single_file(path)
+  if (!single_file(path)) {
```

Comment:
Re: parent folder.

`drive_cp()`: I now remember saying something along these lines: "if user doesn't give instructions about parent folder, we should not even mention it in the request". I still feel this way. So I think we leave it alone.

`drive_mv()` and `drive_mkdir()`: what you say makes sense.